### ISSUE TITLE
NavigationDropdown, SectionedNavigation - Form elements do not have associated labels

### ISSUE THREAD

**Issue by katydecorah** [mapbox/dr-ui#248]
- [ ] The `ControlSelect` in `NavigationDropdown` is missing an associated label.
- [ ] The `ControlInput` in `SectionNavigation` is missing an associated label.

https://web.dev/label/
